src/common/context/__tests__/GlobalContext.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { useContext } from 'react';
+import { AppContextProvider, GlobalContext } from '../GlobalContext';
+import { render, screen, waitFor } from '@testing-library/react';
+import { fetchCustomNetworkId } from '@/components/add-network-form';
+
+jest.mock('@/components/add-network-form', () => ({
+  ...jest.requireActual('@/components/add-network-form'),
+  fetchCustomNetworkId: jest.fn(() => '1111'),
+}));
+
+const GlobalContextTestComponent = () => {
+  const { activeNetwork, networks } = useContext(GlobalContext);
+  return (
+    <div>
+      <div>Global Context Test</div>
+      <div data-testid="activeNetwork">{JSON.stringify(activeNetwork)}</div>
+      <div data-testid="networks">{JSON.stringify(networks)}</div>
+    </div>
+  );
+};
+
+const apiUrls = {
+  mainnet: 'https://api.hiro.so',
+  testnet: 'https://api.testnet.hiro.so',
+};
+
+const getContextField = (fieldId: string) => {
+  return JSON.parse(screen.getByTestId(fieldId).innerHTML);
+};
+
+describe('GlobalContext', () => {
```

Comment:
Added this test just for the new behavior. It can be expanded later to test more of the global context functionality.